packages/app/components/Dropdown.tsx

```diff
@@ -116,34 +116,45 @@
 
```

Comment:
we should probs move this into ui package, not sure why its in components (not your fault, just noticed this)

Reply:
Done that.